keyboards/dz60/keymaps/chrisae9/config.h

```diff
@@ -39,8 +39,14 @@
 #define RGB_DI_PIN E2
 #define RGBLIGHT_ANIMATIONS
 #define RGBLED_NUM 16
-#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_HUE_STEP 1
 #define RGBLIGHT_SAT_STEP 8
-#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_VAL_STEP 16
+
+/* Mouse Config */
+#define MOUSEKEY_DELAY 0
+#define MOUSEKEY_INTERVAL 15
+#define MOUSEKEY_TIME_TO_MAX 10
+#define MOUSEKEY_MAX_SPEED 2.5
 
 #endif
```

Comment:
This file should avoid duplicating the entire `config.h` from the keyboard level.
Instead it should only provide overrides and look sometime similar to:
```c
#pragma once

#undef RGBLIGHT_HUE_STEP
#define RGBLIGHT_HUE_STEP 1
#undef RGBLIGHT_VAL_STEP
#define RGBLIGHT_VAL_STEP 16

/* Mouse Config */
#define MOUSEKEY_DELAY 0
#define MOUSEKEY_INTERVAL 15
#define MOUSEKEY_TIME_TO_MAX 10
#define MOUSEKEY_MAX_SPEED 2.5
```